### PR TITLE
 improvements on glossary links in see-also

### LINKED ISSUE #34451: improvements on glossary links in see-also
## Background

Currently glossary links are mixed in different formats. Label of such links can be improved for:

- in non-glossary entries, to specify that links in the list are points to *Glossary* pages, rather than Guides or References.
- in glossary entries, to clarify that links are points to *Related* terms (against the current glossary page or Glossary mainpage)
- link to /docs/Glossary itself is indeed unnecessary

## Related suggestions

@dipikabh recommends to make the label of all these glossary links more readable (i.e. `Glossary` => `Related glossary terms`):

> Fair enough point about listing the terms. What do you think about preceding the list with a phrase ending in colon, such as "Related glossary terms:". It's a nuance but might be helpful instead of just saying "Glossary".
>
> ```markdown
> - Related glossary terms:
>   - {{Glossary("Term1")}}
>   - {{Glossary("Term2")}}
> ```
>
> - We need to qualify "Glossary" slightly considering we're already on a glossary page.
> - It would be nice to say upfront before the list that the links are glossary terms and not guide or reference pages (so not the last suggestion).
>
> _Originally posted by @dipikabh in https://github.com/mdn/content/pull/34165#discussion_r1655241066 and https://github.com/mdn/content/pull/34095#issuecomment-2194523190_

@bsmth recommends to remove the link of the label (i.e. link points to `/docs/Glossary`)> > This can be merged. Keeping it open for a short time in case we want to apply changes similar to #34165.

> Thanks a lot! I also think the changes look good, with one comment that I think linking to `/Glossary` is not necessary, so I would just prefer to do this:
> 
> ```markdown
> - Glossary
>   - {{Glossary("Term1")}}
>   - {{Glossary("Term2")}}
> ```
> 
> Or indeed
> 
> ```markdown
> - {{Glossary("Term1")}}
> - {{Glossary("Term2")}}
> ```
>
> _Originally posted by @bsmth in https://github.com/mdn/content/pull/34095#issuecomment-2194168271_

### PR DESCRIPTION
**Pull request by PassionPenguin**
fixes #34451

in addition to suggested changes by @bsmth & @dipikabh, I do also corrected the plural form of word `term` in some case when only one glossary entry is in list

~~plus, also edited the writing guideline (24e083580e8bebc736a30fe5abab63334b401d7a) (is the edition correct?)~~

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/what_we_write/index.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Generally, our documentation falls into the following categories:
 
 - Reference
 - Guide
-- Glossary
+- Related glossary terms
```
Comment (dipikabh) [inline files/en-us/mdn/writing_guidelines/what_we_write/index.md:117]: This should stay as is
```suggestion
- Glossary
```

File: files/en-us/web/css/css_grid_layout/index.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ The example below shows a three-column track grid with new rows created at a min
 
 ## See also
 
-- Glossary terms:
+- Related glossary termss:
```
Comment (dipikabh) [inline files/en-us/web/css/css_grid_layout/index.md:142]:
```suggestion
- Related glossary terms:
```

File: files/en-us/web/css/css_transforms/index.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ You can [view this example's source on GitHub](https://github.com/mdn/css-exampl
   - [`<number>`](/en-US/docs/Web/CSS/number)
   - {{cssxref("percentage")}}
   - [`<position>`](/en-US/docs/Web/CSS/position_value)
-- Glossary terms:
+- Related glossary terms:
```
Comment (dipikabh) [inline files/en-us/web/css/css_transforms/index.md:111]: This need not be updated. It is already in a Related concepts section.

File: files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md [hunk 4]
```diff
@@ -42,6 +42,7 @@ add_header 'Access-Control-Allow-Origin' 'origin'
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
+  - {{Glossary("CORS")}}
```
Comment (dipikabh) [inline files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md:46]: Please revert all changes in web/http files

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @PassionPenguin, this change should not be replicated to non-glossary pages; the change on the writing guidelines page is not correct.

Also, we've decided to go with the plural form, that is, "Related glossary terms", even when the list has one item. (This is similar to what we do with the [Examples](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/CSS_property_page_template#examples) section.)

File: files/en-us/web/css/css_cascade/index.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ The opposite also occurs. Sometimes there are no declarations defining the value
 - [Declaring layers](/en-US/docs/Web/CSS/@import#importing_css_rules_into_a_cascade_layer)
   - [named layer](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#the_layer_statement_at-rule_for_named_layers)
   - [anonymous layer](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#the-layer-block-at-rule-for-named-and-anonymous-layers)
-- Glossary: {{glossary("style origin")}}
+- Related glossary terms: {{glossary("style origin")}}
```
Comment (PassionPenguin) [inline files/en-us/web/css/css_cascade/index.md:65]:
```suggestion
- Glossary: {{glossary("style origin")}}
```

File: files/en-us/web/css/css_grid_layout/index.md [hunk 6]
```diff
@@ -139,7 +139,7 @@ The example below shows a three-column track grid with new rows created at a min
 
 ## See also
 
-- Glossary terms:
+- Related glossary terms:
```
Comment (PassionPenguin) [inline files/en-us/web/css/css_grid_layout/index.md:142]:
```suggestion
- Glossary:
```

File: files/en-us/web/css/css_transforms/index.md (hunk 3 above)
Comment (PassionPenguin) [inline files/en-us/web/css/css_transforms/index.md:111]:
```suggestion
- Glossary terms:
```

File: files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md [hunk 7]
```diff
@@ -42,6 +42,7 @@ add_header 'Access-Control-Allow-Origin' 'origin'
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md:45]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md (hunk 4 above)
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsalloworiginnotmatchingorigin/index.md:46]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md [hunk 8]
```diff
@@ -36,5 +36,6 @@ Other possible causes include:
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md:39]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md [hunk 9]
```diff
@@ -36,5 +36,6 @@ Other possible causes include:
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsdidnotsucceed/index.md:40]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsdisabled/index.md [hunk 10]
```diff
@@ -25,5 +25,6 @@ requests will always fail with this error.
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsdisabled/index.md:28]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsdisabled/index.md [hunk 11]
```diff
@@ -25,5 +25,6 @@ requests will always fail with this error.
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsdisabled/index.md:29]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsexternalredirectnotallowed/index.md [hunk 12]
```diff
@@ -30,5 +30,6 @@ thereby avoiding the redirect.
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsexternalredirectnotallowed/index.md:33]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsexternalredirectnotallowed/index.md [hunk 13]
```diff
@@ -30,5 +30,6 @@ thereby avoiding the redirect.
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsexternalredirectnotallowed/index.md:34]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsinvalidallowheader/index.md [hunk 14]
```diff
@@ -31,6 +31,7 @@ ensure that the user agent or HTTP library you're using on the client is up-to-d
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsinvalidallowheader/index.md:34]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsinvalidallowheader/index.md [hunk 15]
```diff
@@ -31,6 +31,7 @@ ensure that the user agent or HTTP library you're using on the client is up-to-d
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsinvalidallowheader/index.md:35]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsinvalidallowmethod/index.md [hunk 16]
```diff
@@ -33,6 +33,7 @@ ensure that the user agent or HTTP library you're using on the client is up-to-d
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsinvalidallowmethod/index.md:36]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsinvalidallowmethod/index.md [hunk 17]
```diff
@@ -33,6 +33,7 @@ ensure that the user agent or HTTP library you're using on the client is up-to-d
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsinvalidallowmethod/index.md:37]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsmethodnotfound/index.md [hunk 18]
```diff
@@ -36,6 +36,7 @@ Make sure your code only uses the permitted HTTP methods when accessing the serv
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsmethodnotfound/index.md:39]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsmethodnotfound/index.md [hunk 19]
```diff
@@ -36,6 +36,7 @@ Make sure your code only uses the permitted HTTP methods when accessing the serv
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsmethodnotfound/index.md:40]:
```suggestion
```

File: files/en-us/web/http/cors/errors/corsmissingallowcredentials/index.md [hunk 20]
```diff
@@ -37,5 +37,6 @@ configuration to set the `Access-Control-Allow-Credentials` header's value to
 ## See also
 
 - [CORS errors](/en-US/docs/Web/HTTP/CORS/Errors)
-- Glossary: {{Glossary("CORS")}}
+- Related glossary term:
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsmissingallowcredentials/index.md:40]:
```suggestion
- Glossary: {{Glossary("CORS")}}
```

File: files/en-us/web/http/cors/errors/corsmissingallowcredentials/index.md [hunk 21]
```diff
@@ -37,5 +37,6 @@ configuration to set the `Access-Control-Allow-Credentials` header's value to
...
+  - {{Glossary("CORS")}}
```
Comment (PassionPenguin) [inline files/en-us/web/http/cors/errors/corsmissingallowcredentials/index.md:41]:
```suggestion
```

**Comment by PassionPenguin** [comment]
@dipikabh i have removed those unrelated changes :) now all changes are restricted within `Glossary/` :)

**Review by dipikabh — APPROVED** [review]
The updates look good. Thanks a lot for your contribution, @PassionPenguin! 🎉

**Pull request merged**